### DISCUSSION TITLE
How to serialize boolean values as true or false, not 1 or 0?

### DISCUSSION THREAD

**Discussion by timsayshey** [wheels-dev/wheels#1263] [Q&A]
I have boolean columns in my database represented by 1s and 0s, and I would like CFWheels to handle them as boolean values (true or false) instead of integers. Specifically, when returning JSON from an action, I want to see true or false rather than 1s and 0s. Is there a built-in way for CFWheels to accomplish this, or do I need to manually loop through the object and convert these values to true or false before serializing it? For example, is there a type attribute in the property() method that allows us to specify the boolean data type, or is there a specific column format I need to set to make CFWheels treat these columns as booleans by convention? Any guidance would be greatly appreciated.

**Comment by chapmandu** [comment]
One possibility it to create a calculated property that returns the value as a Boolean. Whether you create a new property or use the existing name would depend on your use case.

In your User model's config()

property(name="isFancy", sql="CASE WHEN isFancy = 1 THEN 'true' ELSE 'false' END")

This MAY have implications if you use this property in checkboxes. 

HTH
